config/sky-pages/sky-pages.config.js

```diff
@@ -71,6 +71,7 @@ module.exports = {
         },
         command: command,
         componentsPattern: '**/*.component.ts',
+        componentsIgnorePattern: './public/**/*.component.ts',
```

Comment:
I believe `componentsIgnorePattern` will also have to be added to `runtime/config.ts`.  Also, can we be less specific here and just say `componentsIgnorePattner: './public/'`?